src/tract.c

```diff
@@ -323,7 +323,7 @@ int is_read (ITEMSET *iset, FILE *file)
 
 /*--------------------------------------------------------------------*/
 
-int is_recode (ITEMSET *iset, int minfrq, int dir, int *map)
+int is_recode (ITEMSET *iset, int minfrq, int dir, int *map, int mode, int fullS)
```

Comment:
/*Makh2018; Change is_recode function to remove full support items in case of generator itemsets mining */